### DISCUSSION TITLE
Every other fetch system is dying. I hope this one doesn't.

### DISCUSSION THREAD

**Discussion by mobsenpai** [grtcdr/macchina#309] [General]
I've seen most of the popular system fetch tools are unmaintaned for over 6-7 months now atleast.

**Comment by grtcdr** [comment]
Hi,

I just want to clarify that macchina is not unmaintained, the commit history may be perceived as having stagnated especially compared to the first year or two since macchina's release but that's only one side of the coin. 

Although there hasn't been a release in more than a year, I'm regularly updating our dependency list and keeping up with changes in their APIs. I'm also triaging and answering issues for everyone's benefit. Our downstream users are also contributing back to the project, although much more of this is happening in the backend, see @Gobidev's project [pfetch-rs](https://github.com/Gobidev/pfetch-rs) whose existence has benefited both macchina and libmacchina in new features and bug fixes.

It is true however that some pull requests may go stale for longer than they should. I can only do so much with a busy personal, professional and academic life. I'd love to see more of a co-maintenance effort where technically-adept community members contribute back by reviewing stale issues and pull requests. I had a lot of fun interacting with authors of downstream projects because we can all, together, decide the direction of the project.

**Reply by mobsenpai** [reply]
Yeah this one is really good. Keep it up!
